documentation/book/assembly-kafka-bridge-replicas.adoc

```diff
@@ -0,0 +1,21 @@
+// This assembly is included in the following assemblies:
+//
+// assembly-deployment-configuration-kafka-bridge.adoc
+
+// Save the context of the assembly that is including this one.
+// This is necessary for including assemblies in assemblies.
+// See also the complementary step on the last line of this file.
+
+[id='assembly-kafka-bridge-replicas-{context}']
+
+= Replicas
+
+Kafka Bridge clusters can run with a different number of nodes.
+The number of nodes is defined in the `KafkaBridge` resource.
+Running a Kafka Bridge cluster with multiple nodes can provide better availability and scalability.
+However, when running Kafka Bridge on {ProductPlatformName} it is not absolutely necessary to run multiple nodes of Kafka Bridge for high availability.
+When the node where Kafka Bridge is deployed to crashes, {ProductPlatformName} will automatically reschedule the Kafka Bridge pod to a different node.
+However, there may be issues when running Kafka Bridge due to requests being processed by different Kafka Bridge instances.
```

Comment:
These two last lines need to be a lot more clear to explain what is the problem, what does it relate to (consumers), how to prevent it (addressed based routing to the right bridge instance) or how to work around it (setup more independent instances with single replica). This should be also marked as important.

Reply:
IMPORTANT: If a node where Kafka Bridge is deployed to crashes, {ProductPlatformName} will automatically reschedule the Kafka Bridge pod to a different node.
In order to prevent issues arising when client consumer requests are processed by different Kafka Bridge instances, addressed-based routing must be employed to ensure that requests are routed to the appropriate Kafka Bridge instance.
Additionally, each independent Kafka Bridge instance must have a replica. A Kafka Bridge instance has its own state, which is not shared with another instances.